docs/content/preview/yugabyte-voyager/reference/yb-voyager-cli.md

```diff
@@ -396,15 +405,231 @@ The valid *arguments* for import data status are described in the following tabl
+| --ff-db-password <password> | Password to connect to the fall-forward database server |
+| --ff-db-port <port> | Port number of the fall-forward database server (Default: 1521) |
+| --ff-db-schema <schemaName> | Schema name of the fall-forward database |
+| --ff-db-sid <SID> | Oracle System Identifier (SID) that you wish to use while importing data to Oracle instances. Oracle migrations only. |
+| --ff-db-user <username>| Username to connect to the fall-forward database server |
+| --ff-ssl-cert <path>| Fall-forward database SSL Certificate path |
+| --ff-ssl-crl <list>| Fall-forward database SSL Root Certificate Revocation List (CRL) |
+| --ff-ssl-key <Keypath> | Fall-forward database SSL key path |
+| --ff-ssl-mode <SSLmode>| One of `disable`, `allow`, `prefer`(default), `require`, `verify-ca`, or `verify-full`. |
+| --ff-ssl-root-cert | Fall-forward database SSL Root Certificate path |
+| [-h, --help](#command-line-help) | Command line help for setup. |
+| [--oracle-home](#oracle-home) <path> | Path to set $ORACLE_HOME environment variable. `tnsnames.ora` is found in `$ORACLE_HOME/network/admin`. Oracle migrations only.|
+| [--oracle-tns-alias](#ssl-connectivity) <alias> | TNS (Transparent Network Substrate) alias configured to establish a secure connection with the server. Oracle migrations only. |
+| [--parallel-jobs](#parallel-jobs) <connectionCount> | Number of parallel COPY commands issued to the target database. (Default: 1) |
+| [--send-diagnostics](#send-diagnostics) | Send diagnostics information to Yugabyte. (Default: true) |
+| --start-clean | Starts a fresh import with data files present in the `data` directory and if any table on fall- forward database is non-empty, it prompts whether you want to continue the import without the truncating those tables; if yes, then Voyager starts ingesting the data present in the datafiles without upsert mode and for the cases where the table doesn't have a primary key, it may duplicate the data. So, in that case, use `--exclude-table-list` flag to exclude such tables or truncate those tables manually before using the `start-clean` flag. |
+| [--table-list](#table-list) | Comma-separated list of the tables to import data. |
+| [--verbose](#verbose) | Display extra information in the output. |
+| [-y, --yes](#yes) | Answer yes to all prompts during the migration. (Default: false) |
+
+#### Example
+
+```sh
+yb-voyager fall-forward setup --export-dir /path/to/yb/export/dir \
+        --ff-db-host hostname \
+        --ff-db-user username \
+        --ff-db-password password \ # Enclose the password in single quotes if it contains special characters.
+        --ff-db-name dbname \
+        --ff-db-schema schemaName \
+        --parallel-jobs connectionCount
+```
+
+### fall-forward synchronize (Tech preview)
+
+Exports new changes from the target YugabyteDB to import to the fall-forward database so that the fall-forward database can be in sync with the YugabyteDB after cutover.
```

Comment:
```suggestion
Exports new changes from the target YugabyteDB to import to the fall-forward database so that the fall-forward database can be in sync with the YugabyteDB database after cutover.
```